assets/airports/ksea.json

```diff
@@ -231,10 +315,277 @@
+                ["TOU*", "JAWBN*"],
+                ["YVR*", "JAWBN"],
+                ["YYJ*", "ORCUS", "JAWBN"],
+                ["JAWBN", "DIGGN", "ALKIA"]
+            ]
+        },
+        "MARNR6": {
+            "icao": "MARNR6",
+            "name": "Mariner Six",
+            "entryPoints": {
+                "BUHNR": [["BUHNR", "A120+"]],
+                "JIGEB": [["JIGEB", "A120+"]],
+                "MARNR": [],
+                "ROESH": ["ROESH"],
+                "TOU": [["TOU", "A200+"], ["LOOOO", "A170+|S280"]],
+                "YVR": [["YVR", "A200+"], "ROESH"],
+                "YYJ": [["YYJ", "A200+"], ["ORCUS", "A170+|S280"]]
+            },
+            "body": [["MARNR", "A120+|S270"]],
+            "rwy": {
+                "16L": [["BOANE", "A110+|S250"], ["NEHOS", "A80+|S230"], ["GRIFY", "A60|S210"]],
+                "16C": [["BOANE", "A110+|S250"], ["NEHOS", "A80+|S230"], ["GRIFY", "A60|S210"]],
+                "16R": [["BOANE", "A110+|S250"], ["NEHOS", "A80+|S230"], ["GRIFY", "A60|S210"]],
+                "34L": [["PNELA", "A120+|S270"], ["VEGGN", "A150-|S250"], ["UNITT", "A100+|S230"], ["WUBET", "A80+"], ["SHIPZ", "A70|S210"], "EMMSS", "#163"],
+                "34C": [["PNELA", "A120+|S270"], ["VEGGN", "A150-|S250"], ["UNITT", "A100+|S230"], ["WUBET", "A80+"], ["SHIPZ", "A70|S210"], "EMMSS", "#163"],
+                "34R": [["PNELA", "A120+|S270"], ["VEGGN", "A150-|S250"], ["UNITT", "A100+|S230"], ["WUBET", "A80+"], ["SHIPZ", "A70|S210"], "EMMSS", "#163"]
+            },
+            "draw": [
+                ["BUHNR*", "MARNR*"],
+                ["JIGEB*", "MARNR"],
+                ["YVR*", "ROESH*", "MARNR"],
+                ["TOU*", "LOOOO", "MARNR"],
+                ["YYJ*", "ORCUS", "MARNR"],
+                ["MARNR", "BOANE", "NEHOS", "GRIFY"],
+                ["MARNR", "PNELA", "VEGGN", "UNITT", "WUBET", "SHIPZ", "EMMSS"]
+            ]
+        },
+        "OLM2": {
+            "icao": "OLM2",
+            "name": "Olympia Two",
+            "entryPoints": {
+                "BTG": ["BTG", "COUGA"],
+                "OLM": [],
+                "UBG": ["UBG"]
+            },
+            "body": [],
+            "rwy": {
+                "16L": [["OLM", "A170+|S270"], "LACEE", ["ARVAD", "A130|S250"], "FOURT", "#343"],
+                "16C": [["OLM", "A170+|S270"], "LACEE", ["ARVAD", "A130|S250"], "FOURT", "#343"],
+                "16R": [["OLM", "A170+|S270"], "LACEE", ["ARVAD", "A130|S250"], "FOURT", "#343"],
+                "34L": [["OLM", "A130|S250"], "LACEE", "ARVAD", "FOURT", "#070"],
+                "34C": [["OLM", "A130|S250"], "LACEE", "ARVAD", "FOURT", "#070"],
+                "34R": [["OLM", "A130|S250"], "LACEE", "ARVAD", "FOURT", "#070"]
+            },
+            "draw": [
+                ["BTG*", "COUGA", "OLM*"],
+                ["UBG*", "OLM"],
+                ["OLM", "LACEE", "ARVAD", "FOURT"]
+            ]
         }
     },
     "spawnPatterns": [
```

Comment:
in-game: 154acph
real life: 47.05acph

range: 47.05 - 141.15

Please reduce the acph